src/components/List.js

```diff
@@ -88,9 +154,52 @@ You cannot undo this action, and this item's purchase history will be lost.`,
     ];
   }
 
+  function checkmarkColor(result, time) {
```

Comment:
Would you be open to renaming this function? This is definitely not important, but I'm wondering if something like `getCheckboxWithColor` feels more semantic.

Reply:
Defintely!  I think I got lazy having to name so many checkmark things 😂